dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -712,6 +712,22 @@ error.invalidPhaseName=CWWKE0954E: The "{0}" checkpoint phase specified is empty
 error.invalidPhaseName.explanation=An unknown phase was specified as the value of the checkpoint option.
 error.invalidPhaseName.useraction=Review the documentation to determine valid phase names. Reissue the command using a valid checkpoint phase.
 
+error.checkpoint.delete=CWWKE0955E: The server checkpoint request failed because the existing checkpoint image cannot be deleted from the {0} directory.
+error.checkpoint.delete.explanation=A checkpoint request for the server failed because an existing checkpoint image cannot be deleted.
+error.checkpoint.delete.useraction=Ensure the user has permission to delete the existing directory. Reissue the command with a user that has sufficient permissions.
+
+error.checkpoint.workarea.backup=CWWKE0956E: The server checkpoint request failed because the {0} directory cannot be backed up.
+error.checkpoint.workarea.backup.explanation=An error occurred backing up the checkpoint server workarea directory.
+error.checkpoint.workarea.backup.useraction=Ensure the user has sufficient permissions to back up the checkpoint workarea directory. Reissue the command with a user that has sufficient permissions.
+
+error.checkpoint.workarea.restore=CWWKE0957E: The checkpoint server workarea directory cannot be restored from the {0} directory.
+error.checkpoint.workarea.restore.explanation=An error occurred restoring the checkpoint workarea directory.
+error.checkpoint.workarea.restore.useraction=Ensure the user has sufficient permissions to restore the checkpoint workarea. Reissue the command with a user that has sufficient permissions.
+
+info.checkpoint.restore.failed=CWWKE0957I: Restoring the checkpoint server process failed. The server launches normally without using the checkpoint image to restore. Check the {0} log to determine why the checkpoint process was not restored.
+info.checkpoint.restore.failed.explanation=An error occurred restoring a checkpoint server process.  The server will launch normally without using the checkpoint image.
```

Comment:
avoid future tense and use only one space after a period

An error occurred restoring a checkpoint server process. The server launches normally without using the checkpoint image.
or
An error occurred restoring a checkpoint server process.  The server launched normally without using the checkpoint image.

Reply:
This is one case I think we need an exception on for this future tense rule.  This message gets displayed just before we start launching the server "normally" without using the checkpoint.  So the user will see a message:

`... The server launched normally without using the checkpoint image.`

Then immediately followed by a message like:

`Launching defaultServer (Open Liberty 22.0.0.3/wlp-1.0.62.202202080731) on Eclipse OpenJ9 VM, version 11.0.13+8 (en_US)`

I suggest we change to: `Launching the server normally without using the checkpoint image.`

Reply:
Actually we should probably remove the term "normally" because that implies we were launching abnormally before:

`Launching the server without using the checkpoint image.`